### PR TITLE
refactor: shape and post process volume data

### PR DESCRIPTION
**Pull request by wumaolinmaoan**
## Release Notes

- **New Features**
	- Enhanced organization of post-processing settings by restructuring the `globalVolume` object, allowing for more intuitive access to bloom, vignette, color adjustments, and tonemapping settings.

- **Improvements**
	- Updated handling of properties in the `PostProcessVolume` and `ShapeComponent` classes to utilize structured objects for better clarity and maintainability.
	- Improved parameter management in rendering processes, particularly within the `ToneMappingPass`.

- **Bug Fixes**
	- Resolved issues with property access in the `setDatGUI` function to align with the new `globalVolume` structure.

### CODE REVIEW

File: packages/effects-core/src/components/post-process-volume.ts [hunk 1]
```diff
@@ -1,39 +1,54 @@
+import type { Engine } from '..';
+import { spec } from '..';
```
Comment (yiiqii) [inline packages/effects-core/src/components/post-process-volume.ts:2]: 路径问题

**Review by yiiqii — CHANGES_REQUESTED** [review]

File: packages/effects-core/src/render/post-process-pass.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ import type { ShaderWithSource } from './shader';
 import { colorGradingFrag, gaussianDownHFrag, gaussianDownVFrag, gaussianUpFrag, screenMeshVert, thresholdFrag } from '../shader';
 import { Vector2 } from '@galacean/effects-math/es/core/vector2';
 import { Vector3 } from '@galacean/effects-math/es/core/vector3';
+import type { spec } from '..';
```
Comment (yiiqii) [inline packages/effects-core/src/render/post-process-pass.ts:16]: 同上

Comment (wumaolinmaoan) [reply packages/effects-core/src/render/post-process-pass.ts]: 已修改

File: packages/effects-core/src/components/post-process-volume.ts (hunk 1 above)
Comment (wumaolinmaoan) [reply packages/effects-core/src/components/post-process-volume.ts]: 已修改

File: packages/effects-core/src/components/post-process-volume.ts [hunk 3]
```diff
@@ -1,39 +1,54 @@
+import type { Engine } from '..';
```
Comment (yiiqii) [inline packages/effects-core/src/components/post-process-volume.ts:1]: 路径问题

**Review by yiiqii — CHANGES_REQUESTED** [review]

File: packages/effects-core/src/components/post-process-volume.ts (hunk 3 above)
Comment (wumaolinmaoan) [reply packages/effects-core/src/components/post-process-volume.ts]: 已修改

**Review by yiiqii — APPROVED** [review]

**Pull request merged**